site/content/installation/ngf-images/jwt-token-docker-secret.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Getting the NGINX Plus Gateway Fabric Image using JWT
+description: "This document describes how to use a JWT token to get an NGINX Plus Gateway Fabric image from the F5 Docker registry."
+weight: 150
```

Comment:
For the weights in this directory, I think JWT token should be first (100), then Pushing images (200) and then building images (300)